test/jdk/java/nio/file/Files/CreateDirectories.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.io.IOException;
+import java.nio.file.FileAlreadyExistsException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static org.testng.Assert.*;
+import org.testng.SkipException;
+import org.testng.annotations.Test;
+
+/*
+ * @test
+ * @bug 8032220 8293792
+ * @summary Test java.nio.file.Files.createDirectories method
+ * @library ..
+ * @run testng CreateDirectories
+ */
+public class CreateDirectories {
+
+    /**
+     * Creates a symlink which points to a directory that exists. Then calls Files.createDirectories
+     * method passing it the path to the symlink and verifies that no exception gets thrown
```

Comment:
I think you can simplify the wording with something like "Test Files.createDirectories symbolic file with an existing directory."